src/index.js

```diff
@@ -0,0 +1,216 @@
+// curry a function to a specified arity
+const curryToArity = (fn, arity) => {
```

Comment:
[just for the future] need to move this small one to the utils.js or functional.js file

Reply:
@Aleksandr-Panin Good point, I made a fix. Please the check latest commit.

Reply:
https://github.com/BurdaForward/composable-rules/pull/1/commits/747d5a9efd6ccf29283853bc1bb80bc007c76078

Reply:
Looks good :)